projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php

```diff
@@ -538,6 +538,10 @@ public function sort_hidden_submenus() {
 		global $submenu;
 
 		foreach ( $submenu as $menu_slug => $submenu_items ) {
+			if ( empty( $submenu_items ) ) {
```

Comment:
Unnecessary use of `empty` is a pet peeve of mine. We know `$submenu_items` is always set, so this is equivalent to
```suggestion
			if ( ! $submenu_items ) {
```

Reply:
Fair point; they should be equivalent when the var is initialized.